src/website/views/templates/app-layout.pug

```diff
@@ -0,0 +1,934 @@
+---
+title: App layout
+description: A collection of application layout page templates built with Page Title, Buttons, Page-level actions and Footer.
+wide_container: true
+---
+
+.-lumen-only
```

Comment:
If the whole content of the page is lumen only, should we hide the App Layout link from the Templates section in non-lumen themes? It would require minor changes in `/src/website//partials/sidenav.pug`